From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for struct lis3lv02d change
Patch-mainline: Never, kABI workaround
References: git-fixes

The recent addition of init_required field to struct lis3lv02d broke
kABI.

Move the new field into a hole for keeping the kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/misc/lis3lv02d/lis3lv02d.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/misc/lis3lv02d/lis3lv02d.h
+++ b/drivers/misc/lis3lv02d/lis3lv02d.h
@@ -271,9 +271,11 @@ struct lis3lv02d {
 	int			regs_size;
 	u8                      *reg_cache;
 	bool			regs_stored;
-	bool			init_required;
 	u8                      odr_mask;  /* ODR bit mask */
 	u8			whoami;    /* indicates measurement precision */
+#ifndef __GENKSYMS__
+	bool			init_required;
+#endif
 	s16 (*read_data) (struct lis3lv02d *lis3, int reg);
 	int			mdps_max_val;
 	int			pwron_delay;
